docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/AutoStartOrderUtil.java

```diff
@@ -70,6 +86,14 @@
         return new HashSet<>(nodes.values());
     }
 
+    private static boolean isRegularExpressionBased(String[] autoStartContainers) {
+        return autoStartContainers.length == 1;
+    }
+
+    private static boolean isCommaSeparatedBased(String[] autoStartContainers) {
+        return autoStartContainers.length > 1;
```

Comment:
Any chance we will get a regexp that contain a ',' so it's handled as a exact match instead of an expression?